### ISSUE TITLE
Error when initialize ckan with the googleanalytics extension

### ISSUE THREAD

**Issue by miguelbgouveia** [ckan/ckanext-googleanalytics#20]
I install the googleanalytic extension in ckan but when I initialize it throughout the paster serve command I get the following error: 

```
File "/home/ubuntu/workspace/ckan/default/bin/paster", line 11, in <module>
    sys.exit(run())
  File "/home/ubuntu/workspace/ckan/default/local/lib/python2.7/site-packages/paste/script/command.py", line 102, in run
    invoke(command, command_name, options, args[1:])
  File "/home/ubuntu/workspace/ckan/default/local/lib/python2.7/site-packages/paste/script/command.py", line 141, in invoke
    exit_code = runner.run(args)
  File "/home/ubuntu/workspace/ckan/default/local/lib/python2.7/site-packages/paste/script/command.py", line 236, in run
    result = self.command()
  File "/home/ubuntu/workspace/ckan/default/local/lib/python2.7/site-packages/paste/script/serve.py", line 284, in command
    relative_to=base, global_conf=vars)
  File "/home/ubuntu/workspace/ckan/default/local/lib/python2.7/site-packages/paste/script/serve.py", line 329, in loadapp
    **kw)
  File "/home/ubuntu/workspace/ckan/default/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 247, in loadapp
    return loadobj(APP, uri, name=name, **kw)
  File "/home/ubuntu/workspace/ckan/default/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 272, in loadobj
    return context.create()
  File "/home/ubuntu/workspace/ckan/default/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 710, in create
    return self.object_type.invoke(self)
  File "/home/ubuntu/workspace/ckan/default/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 146, in invoke
    return fix_call(context.object, context.global_conf, **context.local_conf)
  File "/home/ubuntu/workspace/ckan/default/local/lib/python2.7/site-packages/paste/deploy/util.py", line 55, in fix_call
    val = callable(*args, **kw)
  File "/home/ubuntu/workspace/ckan/default/src/ckan/ckan/config/middleware/__init__.py", line 45, in make_app
    load_environment(conf, app_conf)
  File "/home/ubuntu/workspace/ckan/default/src/ckan/ckan/config/environment.py", line 97, in load_environment
    p.load_all()
  File "/home/ubuntu/workspace/ckan/default/src/ckan/ckan/plugins/core.py", line 139, in load_all
    load(*plugins)
  File "/home/ubuntu/workspace/ckan/default/src/ckan/ckan/plugins/core.py", line 153, in load
    service = _get_service(plugin)
  File "/home/ubuntu/workspace/ckan/default/src/ckan/ckan/plugins/core.py", line 255, in _get_service
    return plugin.load()(name=plugin_name)
  File "/home/ubuntu/workspace/ckan/default/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2249, in load
    return self.resolve()
  File "/home/ubuntu/workspace/ckan/default/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2255, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/ubuntu/workspace/ckan/default/src/ckanext-googleanalytics/ckanext/googleanalytics/plugin.py", line 6, in <module>
    import genshi
ImportError: No module named genshi
```

I am using the ckan version 2.6.0.

**Comment by camfindlay** [comment]
+1

**Comment by smotornyuk** [comment]
fixed by #21
